Lib9c/Model/State/StateExtensions.cs

```diff
@@ -158,8 +158,7 @@ public static HashSet<T> ToHashSet<T>(this IValue serialized, Func<IValue, T> de
         public static IValue Serialize(this string text) =>
             (Text)text;
 
-        public static string ToString(this IValue serialized) =>
-            ((Text)serialized).Value;
+        public static string ToDotnetString(this IValue serialized) => ((Text) serialized).Value;
```

Comment:
Is there any reason we rename this method?

Reply:
yes, duplicate name of Object.ToString, it cannot be called from outside.